moon/controller.py

```diff
@@ -736,7 +758,10 @@ def try_step_around(self):
     def wait_for_init(self):
         print(self.robot_name, 'Wait for definition of last_position and yaw')
         while self.sim_time is None or self.last_position is None or self.yaw is None:
-            self.update()
+            try:
+                self.update()
+            except:
```

Comment:
this is probably what we discussed, that there are exceptions which you should not catch

Reply:
I am doing, it must not have propagated to this view